### PR TITLE
Remove deprecated chat cli and vl examples

### PR DESCRIPTION
**Pull request by lvhan028**

### CODE REVIEW

**Comment by irexyc** [comment]
should remove https://github.com/InternLM/lmdeploy/blob/c9c225fb3e1aa2cbfa654ec4a49b49d9b2a4fbbd/lmdeploy/turbomind/supported_models.py#L17

**Comment by lvhan028** [comment]
> should remove
> 
> https://github.com/InternLM/lmdeploy/blob/c9c225fb3e1aa2cbfa654ec4a49b49d9b2a4fbbd/lmdeploy/turbomind/supported_models.py#L17

removed

File: lmdeploy/cli/entrypoint.py [hunk 1]
```diff
@@ -11,13 +10,8 @@
...
     """The entry point of running LMDeploy CLI."""
     args = sys.argv[1:]
-    is_deprecated_chat_cli = len(args) > 1 and args[0] == 'chat' and \
-        args[1] in ['torch', 'turbomind']
     CLI.add_parsers()
-    if is_deprecated_chat_cli:
-        SubCliChat.add_parsers()
-    else:
-        CLI.add_parser_chat()
+    CLI.add_parser_chat()
```
Comment (RunningLeon) [inline lmdeploy/cli/entrypoint.py:14]: could move this to CLI.add_parsers() 
https://github.com/InternLM/lmdeploy/blob/c9c225fb3e1aa2cbfa654ec4a49b49d9b2a4fbbd/lmdeploy/cli/cli.py#L276-L280

File: lmdeploy/turbomind/supported_models.py [hunk 2]
```diff
@@ -13,8 +13,6 @@
     InternLMForCausalLM='llama',
     # internlm2
     InternLM2ForCausalLM='internlm2',
-    # internlm-xcomposer
-    InternLMXComposerForCausalLM='llama',
```
Comment (RunningLeon) [inline lmdeploy/turbomind/supported_models.py:17]: ut  failed

**Review by irexyc — APPROVED** [review]

**Review by RunningLeon — APPROVED** [review]
LGTM

**Pull request merged**